charts/steadybit-agent/templates/daemonset.yaml

```diff
@@ -82,6 +94,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: STEADYBIT_HTTP_ENDPOINT_PORT
+              value: {{ .Values.agent.port | quote }}
```

Comment:
Customers might desire to change the port of the agent. In case they do, they would typically do so via the `STEADYBIT_HTTP_ENDPOINT_PORT` env var. I added a more specific configuration option to the chart so that the chart itself can respond to this change (instead of just blindly forward a config value via `env`).